hack/tools/tilt-prepare/main.go

```diff
@@ -230,15 +261,20 @@ func tiltResources(ctx context.Context, ts *tiltSettings) error {
 	// If required, all the task to install cert manager.
 	// NOTE: strictly speaking cert-manager is not a resource, however it is a dependency for most of the actual resources
 	// and running this is the same task group of the kustomize/provider tasks gives the maximum benefits in terms of reducing the total elapsed time.
-	if *certManagerFlag {
+	if ts.CertManager {
 		tasks["cert-manager-cainjector"] = preLoadImageTask(fmt.Sprintf("quay.io/jetstack/cert-manager-cainjector:%s", config.CertManagerDefaultVersion))
 		tasks["cert-manager-webhook"] = preLoadImageTask(fmt.Sprintf("quay.io/jetstack/cert-manager-webhook:%s", config.CertManagerDefaultVersion))
 		tasks["cert-manager-controller"] = preLoadImageTask(fmt.Sprintf("quay.io/jetstack/cert-manager-controller:%s", config.CertManagerDefaultVersion))
 		tasks["cert-manager"] = certManagerTask()
 	}
 
+	kustomizebuild := []string{"clusterctl.crd:./cmd/clusterctl/config/crd/"}
+	for _, j := range ts.DeployObservability {
+		observabilityTool := fmt.Sprintf("%s.observability:./hack/observability/{tool}/", j)
```

Comment:
```suggestion
		observabilityTool := fmt.Sprintf("%s.observability:./hack/observability/%s/", tool, tool)
```
Let's rename `j` to `tool` as well

Reply:
done